src/MultiSelect/Input.js

```diff
@@ -59,6 +60,12 @@ const Input = ({
                     line-height: 16px;
                 }
 
+                .root-input {
+                    max-height: ${inputMaxHeight};
+                    overflow-y: auto;
+                    flex: 1;
+                }
+
```

Comment:
This needs to be pulled out into a separate `<style jsx>` tag because it is dynamic. The `colors.grey900` is not because it can be resolved as a constant.

Reply:
All right, made the requested changes. Let me know if the changes aren't what you meant.